charts/apisix/templates/configmap.yaml

```diff
@@ -165,14 +165,16 @@ data:
     nginx_config:                     # config for render the template to genarate nginx.conf
       error_log: "{{ .Values.logs.errorLog }}"
       error_log_level: "{{ .Values.logs.errorLogLevel }}"         # warn,error
-      worker_rlimit_nofile: 20480     # the number of files a worker process can open, should be larger than worker_connections
+      worker_processes: "{{ .Values.nginx.workerProcesses }}"
+      enable_cpu_affinity: {{ default "true" .Values.nginx.enableCPUAffinity }}
+      worker_rlimit_nofile: {{ default "20480" .Values.nginx.workerRlimitNofile }}     # the number of files a worker process can open, should be larger than worker_connections
       event:
-        worker_connections: 10620
+        worker_connections: {{ default "10620" .Values.nginx.workerConnections  }}
       http:
         enable_access_log: {{ .Values.logs.enableAccessLog }}
         {{- if .Values.logs.enableAccessLog }}
         access_log: "{{ .Values.logs.accessLog }}"
-        access_log_format: "{{ .Values.logs.accessLogFormat }}"
+        access_log_format: '{{ .Values.logs.accessLogFormat }}'
```

Comment:
Why change the quote?

Reply:
single quote allows us not to escape value of `logs.accessLogFormat`, just `'{"remote_addr":"$remote_addr",...}'` not `'{\"remote_addr\":\"$remote_addr\",...}'`

Reply:
@fengxsong After discussions with @spacewander , **we consider the modification here to be a breaking change.**

I will revert the changes here to make it consistent with the APISIX configuration file format.  https://github.com/apache/apisix/blob/7569eb99326f72f69a6e275126decce2b7e486d1/conf/config-default.yaml#L187